content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -93,7 +93,7 @@ delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open
 #### Enable Terra  
  Terra installs SODA On Premise only.  
 - Enable following configurations  
-  - In file installer/ansible/hotpot.yml  update the value  `enable_hotpot : true`.
+  - In file installer/ansible/group_vars/hotpot.yml  update the value  `enable_hotpot : true`.
```

Comment:
Good catch :+1: 
CI seems to be failing though.
Let me try re-running it and check.